docker-compose.build.yml

```diff
@@ -0,0 +1,71 @@
+version: '3'
+services:
+  # Server
+  server_npm:
+    build:
+      context: 'server/'
+      dockerfile: 'Dockerfile.packages'
+    image: liberoadmin/reviewer_server_packages:${IMAGE_TAG}
+    networks: 
+        - 'reviewer_build'
+  server_typescript:
+    build:
+      context: 'server/'
+      dockerfile: "Dockerfile.typescript"
+      args:
+        image_tag: ${IMAGE_TAG}
+    image: liberoadmin/reviewer_server_typescript:${IMAGE_TAG}
+    depends_on:
+      - server_npm
+    networks: 
+        - 'reviewer_build'
+  reviewer_server:
+    build:
+      context: 'server/'
+      dockerfile: "Dockerfile.application"
+      args:
+        image_tag: ${IMAGE_TAG}
+    image: liberoadmin/reviewer_server:${IMAGE_TAG}
+    depends_on:
+      - server_npm
+      - server_typescript
+    networks: 
+        - 'reviewer_build'
+
+  # Client
+  client_npm:
+    build:
+      context: 'client/'
+      dockerfile: 'Dockerfile.packages'
+    image: liberoadmin/reviewer_client_packages:${IMAGE_TAG}
+    networks: 
+        - 'reviewer_build'
+
+  client_webpack:
+    build:
+      context: 'client/'
+      dockerfile: 'Dockerfile.webpack'
+      args:
+        image_tag: ${IMAGE_TAG}
+    image: liberoadmin/reviewer_client_webpack:${IMAGE_TAG}
+    depends_on:
+      - client_npm
+    networks: 
+        - 'reviewer_build'
+
+  reviewer_client:
+    build:
+      context: 'client/'
+      dockerfile: "Dockerfile.application"
+      args:
+        image_tag: "${IMAGE_TAG}"
+    image: liberoadmin/reviewer_client:${IMAGE_TAG}
+    depends_on:
+      - client_npm
```

Comment:
(can probably express only the `client_webpack` dependency, but no harm in doing both)